src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const baseRate = 40;
+  const sevenDaysOff = 50;
+  const threeDaysOff = 20;
```

Comment:
you should take a look at the checklist, there is a point about how you should name your variables

Reply:
I see. ok!